src/ac-power/ac-power.c

```diff
@@ -63,7 +63,7 @@ static int parse_argv(int argc, char *argv[]) {
         return 1;
 }
 
-int main(int argc, char *argv[]) {
+static int _main(int argc, char *argv[]) {
```

Comment:
hmm, not liking the underscore thing. maybe be explicit here: main_internal()? Or maybe call this go() or run()?

(and i also like my bikesheds yellow)

Reply:
I guess `run` is OK.